src/adapter/output/visit_repository_impl.py

```
@@ -7,45 +7,48 @@
             self.db.add(visit_log)
             self.db.commit()
             self.db.refresh(visit_log)
             return visit_log
-        except SQLAlchemyError as e:
+        except SQLAlchemyError:
             self.db.rollback()
             raise HTTPException(
                 status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-                detail=f"An error occurred while working on the database: {str(e)}") from e
+                detail="Internal Server Error - DB Operation Failed"
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)